### PR TITLE
Toggle tooltip on click

### PR DESCRIPTION
**Pull request by jonalport**
@davidgilbertson what do you recommend re the click event on the non-interactive element? Is there a better way to do it?

### CODE REVIEW

**Comment by davidgilbertson** [comment]
I think it's fine. The idea behind a click event on a non-interactive element being 'bad practice' is that screen readers don't read it out as something clickable. But this text isn't accessible to screen readers anyway.

We should probably investigate using `<abbr>` for this with associated `<dl>`, but there's problems with that.

File: src/components/Tooltip/Tooltip.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ import classnames from 'classnames';
 import Portal from '../Portal/Portal';
 import styles from './Tooltip.scss';
 
+// const isTouch = () => typeof Touch === 'object';
+
```
Comment (davidgilbertson) [inline src/components/Tooltip/Tooltip.js]: Commented out code can go

File: src/components/Tooltip/Tooltip.js [hunk 2]
```diff
@@ -56,13 +59,27 @@ class Tooltip extends PureComponent {
...
-      <span // a span so it's valid HTML no matter where it's used
+      // a span so it's valid HTML no matter where it's used
+      // Note:
+      //   the onClick there for touch devices
+      //   but it doesn't seem right to apply role=button
+      //   hence the eslint disabling
+      <span // eslint-disable-line
         onMouseOver={this.showTooltip}
         onMouseOut={this.hideTooltip}
+        onClick={this.toggleTooltip}
```
Comment (davidgilbertson) [inline src/components/Tooltip/Tooltip.js]: Minor: this has the slightly strange behaviour that if you click on desktop, the tooltip will show as soon as you hover, then hide when you click. Not a big deal, but you could instead do:

```js
onTouchStart={this.toggleTooltip}
```

**Review by davidgilbertson — CHANGES_REQUESTED** [review]

File: src/components/Tooltip/Tooltip.js (hunk 1 above)
Comment (davidgilbertson) [reply src/components/Tooltip/Tooltip.js]: FYI, if you were looking at detecting touch, there's only one way to do that that works reliably ... by waiting for the user to touch.

Comment (jonalport) [reply src/components/Tooltip/Tooltip.js]: Yeah decided against it, but failed to delete - whoops!

File: src/components/Tooltip/Tooltip.js (hunk 2 above)
Comment (jonalport) [reply src/components/Tooltip/Tooltip.js]: Yeah I noted that but didn't really think it was a negative thing. Happy to change to that event though.

**Review by davidgilbertson — APPROVED** [review]
Wonderful!

**Pull request merged**